Dockerfile

```diff
@@ -1,28 +0,0 @@
-###############################################################################
-# Copyright 2016-2017 Dell Inc.
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
-#
-# http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-#
-###############################################################################
-# Mongo DB image for EdgeX Foundry
-FROM mongo:4.0-xenial
-MAINTAINER Jim White <james_white2@dell.com>
-
-#copy initialization script for later initialization
-COPY *.js /edgex/mongo/config/
-COPY launch-edgex-mongo.sh /edgex/mongo/config/
-
-#expose Mongodb's port
-EXPOSE 27017
-
-CMD /edgex/mongo/config/launch-edgex-mongo.sh
```

Comment:
Be advised that the Jenkins job will need to be adjusted to look in the new location for the Dockerfiles.

Reply:
Could you give me a clue where the Jenkins files are placed ?

Reply:
We'll need to open an issue in the ci-management repo asking them to adjust the `default` verify job. You can see the build error right now is because of the moved Dockerfiles.